viewing_party/party.py

```diff
@@ -1,23 +1,136 @@
 # ------------- WAVE 1 --------------------
 
+from statistics import StatisticsError, mode
+
+
 def create_movie(title, genre, rating):
-    pass
+    if bool(title) == True and bool(genre) == True and bool(rating) == True:
+        return {
+            "title": title,
+            "genre": genre,
+            "rating": rating
+            }
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data = {

```

Comment:
Similar to above, we want to `append` movie to `user_data["watchlist"]`. What this current line will do is reset the `user_data` dictionary value every time this function is called which we want to avoid especially if we have existing data in it.